core/templates/pages/about-page/about-page.component.ts

```diff
@@ -19,101 +19,76 @@
 import { Component, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 
-import { AboutPageConstants } from './about-page.constants';
+import constants from 'assets/constants';
+import { LoaderService } from 'services/loader.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
+import { UserService } from 'services/user.service';
 import { WindowRef } from
   'services/contextual/window-ref.service';
 
-interface CreditNames {
-  letter: string;
-  names: string[];
-}
-
 @Component({
   selector: 'about-page',
   templateUrl: './about-page.component.html'
 })
 export class AboutPageComponent implements OnInit {
-  aboutPageMascotImgUrl: string;
-  activeTabName: string;
-  allCredits: CreditNames[] = [];
-  listOfNames: string;
-  listOfNamesToThank = [
-    'Alex Kauffmann', 'Allison Barros',
-    'Amy Latten', 'Brett Barros',
-    'Crystal Kwok', 'Daniel Hernandez',
-    'Divya Siddarth', 'Ilwon Yoon',
-    'Jennifer Chen', 'John Cox',
-    'John Orr', 'Katie Berlent',
-    'Michael Wawszczak', 'Mike Gainer',
-    'Neil Fraser', 'Noah Falstein',
-    'Nupur Jain', 'Peter Norvig',
-    'Philip Guo', 'Piotr Mitros',
-    'Rachel Chen', 'Rahim Nathwani',
-    'Robyn Choo', 'Tricia Ngoon',
-    'Vikrant Nanda', 'Vinamrata Singal',
-    'Yarin Feigenbaum'];
-  // Define constant for each tab on the page.
-  TAB_ID_ABOUT = 'about';
-  TAB_ID_FOUNDATION = 'foundation';
-  TAB_ID_CREDITS = 'credits';
-  ALLOWED_TABS = [
-    this.TAB_ID_ABOUT, this.TAB_ID_FOUNDATION, this.TAB_ID_CREDITS];
+  classroomUrl: string;
```

Comment:
Do we still need this to be attached to the component?

Reply:
corrected